release/HOWTO_RELEASE.md

```diff
@@ -41,14 +43,20 @@
 
         mvn versions:set -DnewVersion=2.0.1.Final-SNAPSHOT
 
-* Update `gradle/build.gradle` and `gradle/pom.xml` to next development version
+* Update `gradle-examples/ejb-jaxrs-mail/build.gradle` and `gradle-examples/multi-module/build.gradle` to next development version
```

Comment:
Maybe just say `gradle-examples/*/build.gradle`, in anticipation of other Gradle examples?